solidity/dashboard/src/components/AvailableEthAmount.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { colors } from "../constants/colors"
+
+const AvailableEthAmount = React.memo(({ availableETH }) => {
+  return (
+    <>
+      <span
+        className="text-big text-grey-70"
+        style={{
+          textAlign: "right",
+          padding: "0.25rem 1rem",
+          paddingLeft: "2rem",
+          borderRadius: "100px",
+          border: `1px solid ${colors.grey20}`,
+          backgroundColor: `${colors.grey10}`,
+        }}
+      >
+        {availableETH}
+      </span>
+      <span style={{ color: `${colors.grey60}` }}>&nbsp;ETH</span>
```

Comment:
I think we can `<span className="text-grey-60">&nbsp;ETH</span>` instead of ```<span style={{ color: `${colors.grey60}` }}>&nbsp;ETH</span>```.

Reply:
ah, sure, I can change it, but do we have a rule when to use what?